packages/fiori/src/themes/UploadCollectionItem.css

```diff
@@ -6,6 +6,10 @@
 	display: block;
 }
 
+:host([active][actionable]) {
+	background-color: var(--sapList_SelectionBackgroundColor);
```

Comment:
The list item's background color should not change - only the colour of the texts, buttons and icons inside.
Check if all these look ok when the list is in `[active]` (pressed) mode.

Refer to https://wiki.wdf.sap.corp/wiki/display/visualcore/List+%28Fiori+3%29+-+Interactive+States

Reply:
Done